access-control-app/src/cmr/access_control/services/acl_validation.clj

```diff
@@ -86,6 +86,18 @@
     (:system-identity acl)          :system-identity
     (:catalog-item-identity acl)    :catalog-item-identity))
 
+(defn- group-id-validation
```

Comment:
Thanks for splitting these functions out into named utility functions -- much cleaner!